identity-apps-core/apps/authentication-portal/src/main/webapp/includes/cookie-consent-banner.jsp

```diff
@@ -0,0 +1,170 @@
+            </svg>
+        </div>
+    </div>
+    <div class="banner-content" data-testid="cookie-consent-banner-content">
+        <div class="title" data-testid="cookie-consent-banner-content-title">
+            <%=AuthenticationEndpointUtil.i18n(resourceBundle, "cookie.consent.banner.content")%>
+            <a href="<%= StringEscapeUtils.escapeHtml4(cookiePolicyURL) %>" target="_blank" rel="noopener noreferrer" data-testid="cookie-policy-link">
+                <%=AuthenticationEndpointUtil.i18n(resourceBundle, "cookie.policy")%>
+            </a>.
+        </div>
+    </div>
+    <div class="actions" data-testid="cookie-consent-banner-actions">
+        <button data-testid="cookie-consent-banner-confirm-button" class="ui fluid primary button" type="button" onclick="onCookieConsentClear(this)">
+            <%=AuthenticationEndpointUtil.i18n(resourceBundle, "cookie.consent.banner.clear")%>
+        </button>
+    </div>
+</div>
+<% } %>
+<%
+    if (!StringUtils.isBlank(cookiePolicyURL) && 
```

Comment:
Don't we need to load this through extensions/ since this has nothing to do with the onprem offering?